### PR TITLE
Check text domains in libraries

### PR DESCRIPTION
**Pull request by claudiosanches**
We need to check for text domains in the libraries, there's several errors coming from `action-scheduler`, but Action Scheduler requires some updates in order to us proper fix it, also I think that we should start including the Action Scheduler as a package like we does for product blocks and REST API.

### CODE REVIEW

**Review by rodrigoprimo — COMMENTED** [review]
I agree that we should start checking for i18n errors in the libraries directory and that it makes sense to move Action Scheduler to the packages directory. I suggest that before we merge this PR, we address the errors that `checktextdomain` is reporting to avoid leaving the repository in a broken state with i18n errors. What do you think?

**Comment by claudiosanches** [comment]
@rodrigoprimo I fixed all errors reported by "grunt checktextdomain" command.

File: includes/libraries/action-scheduler/classes/ActionScheduler_Abstract_ListTable.php [hunk 1]
```diff
@@ -107,7 +107,7 @@ abstract class ActionScheduler_Abstract_ListTable extends WP_List_Table {
 	 * `_x` with some default (the package name)
 	 */
 	protected function translate( $text, $context = '' ) {
-		return _x( $text, $context, $this->package );
+		return $text;
```
Comment (rodrigoprimo) [inline includes/libraries/action-scheduler/classes/ActionScheduler_Abstract_ListTable.php]: Could you please clarify this change? What is the point of returning `$text` without any modification? Maybe the idea is to deprecate this function? If so, should we update the docblock?

**Review by rodrigoprimo — APPROVED** [review]
Looks good to me. I'm assuming those changes were submitted to the Action Scheduler repository as well so that they are not overridden the next time we update the library and that the plan is to deprecate ActionScheduler_Abstract_ListTable::translate(). Please let me know if that is not the case.

**Pull request merged**